app/jobs/bulkrax/importer_job.rb

```diff
@@ -3,6 +3,7 @@
 module Bulkrax
   class ImporterJob < ApplicationJob
     queue_as :import
+    require 'csv'
```

Comment:
require in this location will cause reloading issues. it also is almost certainly not nessisary